### PR TITLE
added snackbar

### PR DESCRIPTION
**Pull request by Samarth-5109**
Screenshot (43)

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@import url("../../styles/doc.css");
+.wrapper {
+  display: flex;
+}
+.snackbar {
+  width: 24rem;
+  background-color: #212121;
```
Comment (SRUJANA-PENUGONDA13) [inline components/snackbar/snackbar.css:7]: Use variables for colors

File: components/snackbar/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            </div>
+
+            <div class="type-wrapper btn-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Stacked Snackbar</span>
+                </div> 
+                <!-- <div class="about-type">
+                    <p class="about">There are multiple sizes for square avatars as well. </p> 
+                </div> -->
+                <img class="code-Ss"src="../../images/stackedSnackbar.png" alt="code-shot">
```
Comment (poornima-alugubelly) [inline components/snackbar/snackbar.html:96]: can we remove these commented lines?

File: components/snackbar/snackbar.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
```
Comment (SRUJANA-PENUGONDA13) [inline components/snackbar/snackbar.html:6]: You can keep this in generic or common css files instead of copying in all files.

**Review by MonikaJhingan — APPROVED** [review]
You can use rem consistently in later project, code looks good to be merged.

File: styles/doc.css [hunk 4]
```diff
@@ -1,92 +1,116 @@
...
+  display: flex;
+  align-items: center;
+  /* gap:1rem; */
+  margin-left: 2rem;
+  padding: 1rem;
 }
-.brand_image{
-    width: 3.5rem;
-    height: 3.5rem;
+.brand_image {
```
Comment (poornima-alugubelly) [inline styles/doc.css:42]: display: flex, align-items: center is repeated in both branding and nav-container classes maybe u could make a utilitity class for them and re-use it if u r going to use them many times

File: components/snackbar/snackbar.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+                </div>
+                
+
+            </div>
+
+            <div class="type-wrapper btn-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Stacked Snackbar</span>
+                </div> 
+                <!-- <div class="about-type">
```
Comment (SRUJANA-PENUGONDA13) [inline components/snackbar/snackbar.html:93]: Remove this comments if not required

File: styles/doc.css [hunk 6]
```diff
@@ -1,92 +1,116 @@
...
+.main-container {
+  margin: 1.5rem;
+}
+.types-container {
+  /* display: flex; */
+}
+
+.code-Ss {
+  margin-bottom: 2rem;
+}
```
Comment (poornima-alugubelly) [inline styles/doc.css:133]: better remove the unneccessary commented lines in this code

File: styles/doc.css [hunk 7]
```diff
@@ -1,92 +1,116 @@
...
+
+.type-title {
+  font-size: 1.6rem;
+}
+
+.main-container {
+  margin: 1.5rem;
+}
+.types-container {
+  /* display: flex; */
```
Comment (SRUJANA-PENUGONDA13) [inline styles/doc.css]: remove this as well if not required

**Review by poornima-alugubelly — APPROVED** [review]

File: components/snackbar/snackbar.html [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+          </li>
+          <li class="list-items item-2">
+            <a href=""><i class="fab fa-github github-icon"></i></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="wrapper">
+      <aside class="sidebar-container">
+        <div class="sidebar">
```
Comment (SRUJANA-PENUGONDA13) [inline components/snackbar/snackbar.html:40]: I think this div is not required in the container there is only one element. Once check and remove

**Review by SRUJANA-PENUGONDA13 — APPROVED** [review]
I have added few suggestions once go through it. Keep up good work.

**Pull request merged**